MapboxVision/Services/DataProvider.swift

```diff
@@ -24,11 +24,14 @@ final class RecordedDataProvider: DataProvider {
     
     let dependencies: Dependencies
     let telemetryPlayer: TelemetryPlayer
+    var time: UInt
     
     init(dependencies: Dependencies) {
         self.dependencies = dependencies
         self.telemetryPlayer = TelemetryPlayer()
         telemetryPlayer.read(fromFolder: dependencies.recordingPath.recordingPath)
+        telemetryPlayer.scrollData(Int(dependencies.startTime))
```

Comment:
This line may be moved to `start` method in order to scroll to recording's start time each playback start.